mmv1/products/dialogflowcx/Flow.yaml

```diff
@@ -27,17 +27,35 @@ timeouts: !ruby/object:Api::Timeouts
   update_minutes: 40
 custom_code: !ruby/object:Provider::Terraform::CustomCode
   custom_import: templates/terraform/custom_import/dialogflowcx_flow.go.erb
-  pre_create: templates/terraform/pre_create/dialogflow_set_location.go.erb
+  pre_create: templates/terraform/pre_create/dialogflowcx_set_location_skip_default_obj.go.erb
```

Comment:
I like how making the create action swap to performing an update action helps pull the resource into state!

Reply:
Thanks; I'm glad it worked and I didn't have to recreate it by hand.